features/step_definitions/assertions_steps.rb

```diff
@@ -2,10 +2,11 @@
     expect(page).to have_content content
 end
 
-Then("I should be on the landing page") do
-    expect(current_path).to eq root_path 
-end
-
 Then("I should not see {string}") do |content|
     expect(page).to have_no_content content
-end  
\ No newline at end of file
+end  
+
+Then("I should be on the landing page") do
+    visit root_path
```

Comment:
`expect(current_path).to eq root_path` 
This is an assertion step

Reply:
hm.. isn't it already in that folder..?

Reply:
When you say `visit root_path` you are telling it to go to the root_path.
With `expect(current_path).to eq root_path` you are checking if you are on the root_path, there is  a big difference in those two things.

Reply:
done